src/fulltrade/BarrierCard.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes, Component } from 'react';
+import BarrierInput from './BarrierInput';
+
+export default class BarrierCard extends Component {
+    static propTypes = {
+        barrier: PropTypes.number,
+        barrier2: PropTypes.number,
+        barrier1Info: PropTypes.shape({
```

Comment:
barrier is the actual form params to be used for price proposal ~
barrierInfo.value is some value suggested from backend, I'll name it defaultValue since I confuse you